src/components/templates/PartyMap/components/Map/Map.tsx

```diff
@@ -201,32 +179,40 @@ export const Map: React.FC<PropsType> = ({
     onSeatClick,
   });
 
+  // TODO: this probably doesn't even need to be a hook.. it's more of a component if anything. We can clean this up later
   const partygoersOverlay = usePartygoersOverlay({
     showGrid,
     userUid,
     venueId,
     withMiniAvatars: venue.miniAvatars,
-    rows,
-    columns,
+    rows: totalRows,
+    columns: totalColumns,
     partygoers,
     setSelectedUserProfile,
   });
 
   const roomOverlay = useMemo(
     () =>
-      venue.rooms.map((room) => (
-        <PartyMapRoomOverlay
-          key={room.title}
+      venue.rooms.map((room, index) => (
```

Comment:
Does this build now `rooms` is optional?

Reply:
I would expect not.

Reply:
This has been resolved now.